.gitignore

```diff
@@ -35,3 +35,7 @@ test.txt
 !/.idea/modules.xml
 !/.idea/runConfigurations/
 !/.idea/scopes/
+
+# Helm
+*.tgz
+*.lock
```

Comment:
Since this is chart specific, could go into a gitignore file in the charts directory.

Reply:
Fixed.